### ISSUE TITLE
D2 Gauss Ammo Distribution when killed

### ISSUE THREAD

**Issue by StrifeinCA** [CDarrow/DXX-Retro#19]
[Issue]
When killing a player that has picked up multiple Gauss ammo packs, and is killed, the ammo stays in that dropped cannon.  It does not send the ammo packs back into the game.  The player that was just killed can pick up that same cannon, and have all 20k Gauss available to them.  The player that did the killing can not collect on any of the gauss powerups, as the ammo packs do not drop.  The only option is to drop your cannon, and pick their full cannon up.  This presents itself as an issue because if a game last long enough, everyone will be left with just the 2500 rounds that come with the gun.

[Expected results]
When killed, the cannon should be dropped and if picked back up only have 2500 ammo.  You need to pick up the ammo boxes through out the level that have been re-spawned.

[Tested with Tyranny]
- Ty picked up all Gauss ammo(20,000) and a Gauss cannon in the level leaving me no ammo other than my cannon stock levels(2500).
  - I killed him and no ammo boxes dropped, just his Gauss Cannon.
  - I was unable to collect any ammo by flying over his Cannon either.
  - Ty picked up his Cannon and had all 20,000 still in it.
  - He dropped his Cannon and I dropped mine.
  - I picked his up, and had all 20,000 rounds
  - Ty picked up mine and was left with the default 2500.

**Comment by roncli** [comment]
@CDarrow I'm guessing we can close this?

**Comment by CDarrow** [comment]
Nope.  I thought we had consensus in D2, but the original behavior is desired as an option, and I haven't done that yet.
